spec/spec_helper.rb

```diff
@@ -112,7 +112,10 @@ def only(*whitelist)
   end unless method_defined?(:only)
 end
 
-# set default time zone in TZ environment variable
-# which will be used to set session time zone
-ENV['TZ'] ||= 'Europe/Riga'
-# ENV['TZ'] ||= 'UTC'
+# Set Oracle Database session timezone
+# this setting needs to be in place to make sure that Oracle properly represents
+# the timestamp format on your local machine
+# Do not use 'OS_TZ' value or fixed offset values like '05:00', for daylight saving-enabled timezones
+# The setting cannot be derived directly from Windows operating system or ruby, as on the Windows machines,
+# timezone names are non-standard ones and are subject to translations to different languages.
+ENV['ORA_SDTZ'] ||= 'Europe/London'
```

Comment:
OK... Oracle by default (or with OS_TZ) is using non-DST enabled offset (works that way on Linux too). So seems we really need to override it if our OS timezone is DST-enabled.

Something else is flawed (maybe in ruby-oci8) that setting/getting timestamp is not symmetric but that's a subject of separate investigation (out of scope for this PR).